README.adoc

```diff
@@ -405,9 +405,9 @@ all of its lines.
 Uncomment all the lines in the [hotspot file=0]`openapi.yaml` file.
 
 openapi.yaml
-[source, text, linenums, role="code_column"]
+[source, text, linenums, role='code_column']
 ----
-include::finish/src/main/webapp/META-INF/openapi.yaml[tags=**]
+include::finish/src/main/webapp/META-INF/openapi.yaml[]
 ----
 
 Since this document is complete, you can also set the `mp.openapi.scan.disable` property
```

Comment:
`mp.openapi.scan.disable` should be a hotspot

Reply:
Line 414, if the file exists, the reference to the file should be a hotspot and not just surrounded in back ticks

Reply:
Master branch does not have the suggested as hotspots and seems like the scenario here does not require them: from `public`
![Uploading image.png…]()